pose-detection/src/calculators/tracker.md

```diff
@@ -0,0 +1,58 @@
+# Trackers
+
+The purpose of *detection* is to locate objects in a frame; but there is no
+association of objects from one frame to the next. To understand whether object
+*k* in frame *t* corresponds to a detected object in frame *t-1*, we must employ
+a *tracker*.
+
+Why would we want a tracker? Well, in a multi-person detection setting, it is
+often useful to keep track of who is who. Imagine an application where two
+people are competing in a live jumping jack competition. Simply detecting both
+individuals in each frame is not sufficient for the application, since we must
+maintain the identity of each detection and accumulate state (in this case the
+number of jumping jacks).
+
+A tracker is an algorithm that (typically) runs downstream of detection. The
+tracker maintains a history of objects (i.e. *tracks*), and its objective is to
+*link* incoming detections with these tracks (and spawn new tracks or kill old
+ones).
+
+To adequately match incoming detections with preexisting tracks, we must have a
+measure that scores the similarity:
+
+<img src="https://latex.codecogs.com/png.image?\dpi{300}&space;\mathrm{score}&space;=&space;\mathrm{similarity\_fn}(\mathrm{detection},&space;\mathrm{track})" title="\mathrm{score} = \mathrm{similarity\_fn}(\mathrm{detection}, \mathrm{track})" width="300"/>
```

Comment:
I'm not familiar with including math in HTML like this, but can we be sure this link never breaks? And did you verify that the latex in the title actually works?